src/Workspaces/Core/Portable/Editing/SyntaxGenerator.cs

```diff
@@ -1395,34 +1395,42 @@ protected static SyntaxList<TNode> RemoveRange<TNode>(SyntaxList<TNode> list, in
         /// <summary>
         /// Creates a statement that declares a single local variable.
         /// </summary>
-        public abstract SyntaxNode LocalDeclarationStatement(SyntaxNode type, string identifier, SyntaxNode initializer = null, bool isConst = false);
+        public abstract SyntaxNode LocalDeclarationStatement(
+            SyntaxNode type, string identifier, SyntaxNode initializer = null, bool isConst = false);
+
+        internal abstract SyntaxNode LocalDeclarationStatement(
+            SyntaxNode type, SyntaxToken identifier, SyntaxNode initializer = null, bool isConst = false);
 
         internal abstract SyntaxNode WithInitializer(SyntaxNode variableDeclarator, SyntaxNode initializer);
         internal abstract SyntaxNode EqualsValueClause(SyntaxToken operatorToken, SyntaxNode value);
 
         /// <summary>
         /// Creates a statement that declares a single local variable.
         /// </summary>
-        public SyntaxNode LocalDeclarationStatement(ITypeSymbol type, string name, SyntaxNode initializer = null, bool isConst = false)
-        {
-            return LocalDeclarationStatement(TypeExpression(type), name, initializer, isConst);
-        }
+        public SyntaxNode LocalDeclarationStatement(
+            ITypeSymbol type, string name, SyntaxNode initializer = null, bool isConst = false) =>
+                LocalDeclarationStatement(TypeExpression(type), name, initializer, isConst);
 
         /// <summary>
         /// Creates a statement that declares a single local variable.
         /// </summary>
-        public SyntaxNode LocalDeclarationStatement(string name, SyntaxNode initializer)
-        {
-            return LocalDeclarationStatement((SyntaxNode)null, name, initializer);
-        }
+        public SyntaxNode LocalDeclarationStatement(string name, SyntaxNode initializer) =>
+            LocalDeclarationStatement((SyntaxNode)null, name, initializer);
```

Comment:
IDE preference is to put the => on the next line (indented) for multi-line expression bodied members.